frontend/src/store/slices/auth/auth.slice.ts

```diff
@@ -0,0 +1,38 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { ReducerName, DataStatus, StorageKey } from 'common/enums';
+import { AppThunk } from 'common/types';
+import { IUserLoginPayload } from 'healthcare-shared/common/interfaces';
+import { authApi, storage } from 'services';
+
+type AuthState = {
+  user: Record<string, unknown> | null; // TODO: change to User type;
+  isAuthorized: boolean;
+  dataStatus: DataStatus;
+};
```

Comment:
```suggestion
type AuthState = {
  user: Record<string, unknown> | null; // TODO: change to User type;
  dataStatus: DataStatus;
};
```
We don't need it because we can calculate it this way - `Boolean(user)` in places where we need it